src/components/Button/src/Button.vue

```diff
@@ -230,6 +294,15 @@ export default {
 				textColor: this.textColor,
 			});
 		},
+		iconComponent() {
```

Comment:
I don't think it's a good idea to hardcode these icons as part of the variants themselves. If we cannot pass the icons through the default slot for some reason I'd suggest we push back on design here and ask if it's alright to just change the color of the button without also prepending the prefix icons.